### PR TITLE
Create ytdl-sub.xml (#393)Co-authored-by: Roxedus <me@roxedus.dev>

### PR DESCRIPTION
**Pull request by cellardoor452**
Template for YTDL-SUB. None exist currently. https://github.com/jmbannon/ytdl-sub/issues/424

### CODE REVIEW

File: templates/ytdl-sub.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <Support>https://github.com/jmbannon/ytdl-sub/wiki</Support>
+  <Project>https://github.com/jmbannon/ytdl-sub</Project>
+  <Overview>Automate downloads and metadata generation with YoutubeDL&#xD;
+&#xD;
+&#xD;
+Please take your time to go through the walkthrough&#xD;
+&#xD;
+https://github.com/jmbannon/ytdl-sub/wiki</Overview>
+  <Category>Downloaders: MediaApp:Video</Category>
+ <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/ytdl-sub<TemplateURL/>
```
Comment (Roxedus) [inline templates/ytdl-sub.xml:18]: missing file extension

File: templates/ytdl-sub.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  <Icon/>
+  <ExtraParams/>
+  <PostArgs/>
+  <CPUset/>
+  <Requires/>
```
Comment (Roxedus) [inline templates/ytdl-sub.xml:23]: Unnecessary fluff
```suggestion
```

**Review by cellardoor452 — COMMENTED** [review]
Accidentally uploaded a template under this reuqest

**Comment by cellardoor452** [comment]
> reuqest

request

**Review by Roxedus — APPROVED** [review]

**Pull request merged**